### DISCUSSION TITLE
Who can merge a PR

### DISCUSSION THREAD

**Discussion by tobolar** [modelica/ModelicaStandardLibrary#3719] [Q&A] [answered]
As the positions ot the project leader and project deputy of this MAP-LIB are vacant at the moment, I wonder who is in charge to merge the pull requests which passed all the checks.
IMO, anyone from MAP-LIB or package officers seem reasonable.

**Comment by AHaumer** [comment]
AFAIK every library officer, but you have to wait for the automatic tests and the approvals.
An hour ago I just merged one of my PRs.

**Comment by beutlich** [comment, marked as answer]
In my ideal world, the assignee of the PR merges the changes. Every member of MAP-Lib can do so. Would be also ideal, if the open reviewers are unassigned.

**Reply by tobolar** [reply]
A question remains who shall be the assignee?

**Reply by beutlich** [reply]
That is the freedom of the PR author. There is no strict process here.

**Comment by tobolar** [comment]
Do you mean "reviewes"?

**Comment by beutlich** [comment]
No, I usually remove the reviewers who did not review from merged PRs.

![grafik](https://user-images.githubusercontent.com/14896695/104726323-b8c81100-5733-11eb-975d-2688623d8f30.png)

**Reply by tobolar** [reply]
Ok, having open reviewers after merging PR makes no sense.
